ops/services/postgres_db/monitor.tf

```diff
@@ -3,9 +3,8 @@ resource "azurerm_monitor_diagnostic_setting" "postgres" {
   target_resource_id         = azurerm_postgresql_flexible_server.db.id
   log_analytics_workspace_id = var.log_workspace_id
 
-  log {
+  enabled_log {
     category = "PostgreSQLLogs"
-    enabled  = true
```

Comment:
is this enabled by default?

Reply:
AFAIK They are not enabled by default. Due to changes to the provider, we can add the `enabled_log` block to make this more explicit.

Reply:
so since its not enabled by default, will removing the `enabled=true` turn this block off?

Reply:
This works in tandem with the changed `enabled_log` two lines up. This log will continue to be enabled.